create-api-docs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+mkdir tempgopath
+
+orig_path=$(pwd)
+
+export GOPATH="$(pwd)/tempgopath"
+export GO111MODULE="on"
+
+mkdir -p "$GOPATH/src/github.com/projectsyn"
+
+ln -s "$(pwd)" "$GOPATH/src/github.com/projectsyn/lieutenant-operator"
+
+cd "$GOPATH/src/github.com/projectsyn/lieutenant-operator" || return 1
```

Comment:
Could the `return` be done instead by using `set -euo pipefail` at the beginning?

Reply:
Yes, that should have the same effect. Adding that.